README-ja.md

```diff
@@ -118,13 +137,28 @@
 
 - `{`...`}`を使った中括弧展開によって、似たようなコマンドを複数回入力しなくて済む。例えば、 `mv foo.{txt,pdf} some-dir` (両方のファイルを移動させる), `cp somefile{,.bak}` (`cp somefile somefile.bak` と展開される)、`mkdir -p test-{a,b,c}/subtest-{1,2,3}` (すべての可能な組み合わせでディレクトリが作られる).
 
+- 展開の順序は括弧→チルダ、パラメータや変数、計算機号、コマンド置換(左から右)→文字の分割→ファイル名の順だ。(例えば、{1..20}のような範囲は{$a..$b}というようには表現できない。seqやforループを使ってこんな風に表すことができる。seq $a $b or for((i=a; i<=b; i++)); do ... ; done
```

Comment:
How about quoting those commands and the snippet with backquotes like `seq` or `seq $a $b or for((i=a; i<=b; i++)); do ... ; done`?